lib/utils/runTestUtils.js

```diff
@@ -0,0 +1,15 @@
+function checkMode(htmlInput, htmlPreview) {
```

Comment:
Maybe let's rename the function to something mentioning html: `checkHtmlMode`

Reply:
Will rename to checkRenderMode (cause this might be rendering PDFs in the future too)